Tests/LibWeb/Ref/reference/webkit-compat.html

```diff
@@ -0,0 +1,9 @@
+<style>
+    * {
+        margin: 0;
+    }
+    body {
+        background-color: white;
+    }
+</style>
+<img src="./images/webkit-compat.png">
```

Comment:
Same as for the other PR, if this is identical to `box-shadow`, then please use that in the reference page instead of a screenshot. Screenshots are a lot bigger and require updating when rendering changes.

Also, this is named like it tests all of them, but only tests `-webkit-box-shadow`.

Reply:
Renamed and changed to avoid images